pom.xml

```diff
@@ -611,10 +611,11 @@
                     <artifactId>maven-surefire-plugin</artifactId>
                     <configuration>
                         <!-- locale settings must be set on the command line before startup -->
-                        <argLine>-Duser.language=en -Duser.country=US</argLine>
+                        <argLine>-Duser.language=en -Duser.country=US -Dfile.encoding=UTF-8 -Duser.timezone=UTC -Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager</argLine>
```

Comment:
timezone is already set as part of the global test configuration, I don't think we need to add them here.